arch/ecmwf/hpc2020/nvhpc/22.11/env.sh

```diff
@@ -24,10 +24,12 @@ module_unload() {
 module purge
 
 # Load modules
+module_load cmake/3.25.2
```

Comment:
CMake 3.28 is loaded further down
```suggestion
module_load cmake/3.25.2
```

Reply:
Indeed. I have removed 3.25.2 from the list.